pyvista/core/objects.py

```diff
@@ -357,23 +357,126 @@ def get_data_range(self, arr=None, preference='row'):
 
 
 class Texture(_vtk.vtkTexture, DataObject):
-    """A helper class for vtkTextures."""
+    """A helper class for vtkTextures.
 
-    def __init__(self, *args, **kwargs):
+    Parameters
+    ----------
+    uinput : str, vtkImageData, vtkTexture, list, optional
+        Filename, ``vtkImagedata``, ``vtkTexture``, or a list of images to
+        create a cubemap. If a list of images. Must be of the same size and in
+        the following order:
+
+        * -X
+        * -Y
+        * -Z
+        * +X
+        * +Y
+        * +Z
```

Comment:
From a botched test it seemed to me that this order is incorrect. After that I could get the correct mapping using the order used in our `download_sky_box_cube_map()` example or the `test_cubemap()` test: ` ['posx', 'negx', 'posy', 'negy', 'posz', 'negz']`. We should double-check this.

Well, I checked with one of the doctests. This does need to change.

Reply:
You're correct. Order listed here is incorrect. Example in this docstring is correct.